cli/fs_util.rs

```diff
@@ -698,7 +702,34 @@ mod tests {
     ]
     .iter()
     .map(|f| ModuleSpecifier::parse(f).unwrap())
-    .collect::<Vec<ModuleSpecifier>>();
+    .collect::<Vec<_>>();
+
+    assert_eq!(result, expected);
+
+    let scheme = if cfg!(target_os = "windows") {
+      "file:///"
+    } else {
+      "file://"
+    };
+    let result = collect_specifiers(
+      vec![format!(
+        "{}{}",
+        scheme,
+        root_dir_path.join("child").to_str().unwrap()
```

Comment:
I think backslashes should be replaced with forward slashes here for windows? Otherwise it will be testing `file:///C:\something\other\`.

Reply:
Wouldn't it have conflicts with files with "\" in their name?